tests/ignite/contrib/metrics/test_precision_recall_curve.py

```diff
@@ -173,13 +171,22 @@ def _test(y_pred, y, batch_size, metric_device):
         assert precision_recall_curve(np_y, np_y_pred)[2] == pytest.approx(res[2].cpu().numpy())
 
     def get_test_cases():
+        torch.manual_seed(12 + rank)
         test_cases = [
             # Binary input data of shape (N,) or (N, 1)
-            (torch.randint(0, 2, size=(10,)), torch.randint(0, 2, size=(10,)), 1),
-            (torch.randint(0, 2, size=(10, 1)), torch.randint(0, 2, size=(10, 1)), 1),
+            (torch.randint(0, n_classes, size=(10,)), torch.randint(0, n_classes, size=(10,)), 1),
+            (torch.randint(0, n_classes, size=(10, 1)), torch.randint(0, n_classes, size=(10, 1)), 1),
```

Comment:
IMO, there is no point to introduce `n_classes` for binary data, let's keep `randint(0, 2`